client/src/components/singleBox.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+
+
+console.log('hello from singlebox');
+const SingleBox = (props) => {
+  console.log('this is PROPS FROM SINGLEBOX', props.info);
+  console.log(props.info[1].name);
+ 
+  let individualBox = props.info;
+  
+  
+
+  console.log('!!!', props.info);
```

Comment:
It's good practice to clear out the leftover console logs when you're done with them 👍